index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+</head>
+
+<body>
+    <main id="main">
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <div id="img-caption">
+            <p>The man who saved a billion lives</p>
+        </div>
```

Comment:
Please put this element inside a <header> tag, because  Tags have semantic meanings different from the <main> tag thus what is supposed to be in the header tag should not be placed in the main tags.